sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseService.java

```diff
@@ -876,4 +903,129 @@ private Predicate createNewCaseFilter(CriteriaBuilder cb, From<?, Case> caze, Da
 
 		return newCaseFilter;
 	}
+
+	public Case getRelevantCase(Person person, Disease disease, Date referenceDate) {
```

Comment:
Should probably also be called "getRelevantCaseForFollowUp" if it's not also used in another context. What exactly is this method used for?

Reply:
OK